### DISCUSSION TITLE
What are the HSL color values for the primary theme in FumaDocs Core?

### DISCUSSION THREAD

**Discussion by kakarj** [fuma-nama/fumadocs#1077] [General]
I really liked the purple shades and would like to know the exact values.
<img width="273" alt="Screenshot 2024-11-20 at 8 07 21 AM" src="https://github.com/user-attachments/assets/dffa2ff2-31dd-4520-96cd-ed5446a9c409">

**Comment by fuma-nama** [comment]
See https://github.com/fuma-nama/fumadocs/blob/05d224c72ae314ee9b08c51a054213d08a248921/apps/docs/app/global.css#L24

**Comment by kakarj** [comment]
thanks <3
